src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -229,35 +229,48 @@ def load_params(command):
     _update_command_definitions(command_table)
 
 
-def get_command_table():
+def get_command_table(module_name=None):
     '''Loads command table(s)
+    When `module_name` is specified, only commands from that module will be loaded.
+    If the module is not found, all commands are loaded.
     '''
-    installed_command_modules = []
-    try:
-        mods_ns_pkg = import_module('azure.cli.command_modules')
-        installed_command_modules = [modname for _, modname, _ in
-                                     pkgutil.iter_modules(mods_ns_pkg.__path__)]
-    except ImportError:
-        pass
-    logger.debug('Installed command modules %s', installed_command_modules)
-    cumulative_elapsed_time = 0
-    for mod in installed_command_modules:
+    loaded = False
+    if module_name and module_name != 'acs':
```

Comment:
If a module is there, we don't want to assume it is there and try to load it.
Also, we still need to load all module occasionally for example like when a user runs `az`.